tests/models/bark/test_modeling_bark.py

```diff
@@ -531,8 +532,11 @@ def prepare_config_and_inputs_for_common(self):
 # Need this class in oder to create tiny model for `bark`
 # TODO (@Yoach) Implement actual test methods
 @unittest.skip("So far all tests will fail.")
```

Comment:
Should this be removed?

Reply:
The original motivation to have this `BarkModelTest` is so we can create the tiny model for `bark`. However, @sanchit-gandhi mentioned 

> Bark is irregular in the sense that it's a concatenation of three auto-regressive models, meaning there's no notion of a forward pass

see [here](https://github.com/huggingface/transformers/pull/25372#discussion_r1287159014), and this `BarkModelTest` doesn't make sense.

I will remove this test class.